<ui version="4.0" >
 <class>OptionEditor</class>
 <widget class="QWidget" name="OptionEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptionEditor</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize" >
      <bool>false</bool>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="lblType" >
         <property name="text" >
          <string>Type :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="typesTable" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="rowCount" >
          <number>0</number>
         </property>
         <column>
          <property name="text" >
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="valueBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Value</string>
      </property>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget" >
         <property name="currentIndex" >
          <number>3</number>
         </property>
         <widget class="QWidget" name="pageValue" >
          <layout class="QVBoxLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lblValue" >
             <property name="text" >
              <string>Value :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineValue" />
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageIp" >
          <layout class="QVBoxLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lblAddress" >
             <property name="text" >
              <string>IP Address :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineAddress" >
             <property name="inputMask" >
              <string>000.000.000.000;_</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageFlag" >
          <layout class="QVBoxLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lblFlag" >
             <property name="text" >
              <string>Flag :</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <widget class="QRadioButton" name="radioTrue" >
               <property name="text" >
                <string>True</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioFalse" >
               <property name="text" >
                <string>False</string>
               </property>
               <property name="checked" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="StringListEditor" name="pageStringList" />
         <widget class="AddressListEditor" name="pageAddressList" />
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>AddressListEditor</class>
   <extends>QWidget</extends>
   <header>addresslisteditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StringListEditor</class>
   <extends>QWidget</extends>
   <header>stringlisteditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
